client/my-sites/theme/themes-related-card/index.jsx

```diff
@@ -70,4 +71,8 @@ const ThemesRelatedCard = React.createClass( {
 	}
 } );
 
-export default ThemesRelatedCard;
+export default connect(
+	state => ( {
+		getDetailsUrl: getThemeDetailsUrl.bind( null, state )
```

Comment:
Yes, that's actually a lot better. (I've been using that style in some other PRs anyway, not sure why I didn't here).